services/core-web/src/components/mine/Compliance/ComplianceOrdersTable.js

```diff
@@ -71,6 +71,15 @@ const fileColumns = [
   },
 ];
 
+const transformFileRowData = (file, inspectionId) => ({
```

Comment:
This `const` is already declared earlier in this file, its causing issues on the build pipeline.